src/collective/lazysizes/transform.py

```diff
@@ -141,7 +141,7 @@ def _blacklist(self, result, blacklisted_classes):
 
         path = []
         for css_class in blacklisted_classes:
-            path.append('{0}{1}//img|{0}{1}//iframe|{0}{1}//blockquote'.format(
+            path.append('{0}{1}|{0}{1}//img|{0}{1}//iframe|{0}{1}//blockquote'.format(
```

Comment:
this change needs a test.

Reply:
I can provide a test.
The interface description for ``css_class_blacklist`` says:
""" [...] elements with that class directly applied to them, or to a parent element, will be skipped."""
https://github.com/collective/collective.lazysizes/blob/master/src/collective/lazysizes/interfaces.py#L23

I actually also expected the blacklist class being able to set on the img elements directly, even before reading the interface description :)